recipes/sktime/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "sktime" %}
+{% set version = "0.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: f066ce462b5f6013391ec7697471a22e6f67af70b85e670202fed02ac87488b1
+
+build:
+  number: 0
+  noarch: python
```

Comment:
If this package is compiling c-extensions, then you should remove `noarch: python` since it needs to compile different code for different platforms